lib/conjur/trusted_proxy_filter.rb

```diff
@@ -10,8 +10,8 @@ module Conjur
   #
   # Example: TRUSTED_PROXIES=4.4.4.4,192.168.100.0/24
   class TrustedProxyFilter
-    def initialize(env: ENV, options: { disable_cache: true })
-      @env = env
+    def initialize(config: Conjur::ConjurConfig.new, options: { disable_cache: true })
+      @config = config
```

Comment:
Re: `options: { disable_cache: true }`: Is it likely we'll have other options in the future?  Otherwise perhaps we should YAGNI and just change the signature to:
```ruby
def initialize(config: Conjur::ConjurConfig.new, disable_cache: true)
```

Reply:
Sounds good, yeah.